components/automate-gateway/gateway/server.go

```diff
@@ -450,6 +450,9 @@ func (s *Server) startHTTPServer() error {
 	// custom mux route for export of all reports for a single node
 	mux.HandleFunc("/api/v0/compliance/reporting/node/export", s.NodeExportHandler)
 
+	// custom mux route for fetching report details of a single node
+	mux.HandleFunc("/api/v0/compliance/reporting/nodeheader/id/", s.NodeFetchHandler)
```

Comment:
Why are we following this approach to make it part of http server? @Dmaddu @mchengat88
We should be following generic way of using GRPC service and behind IAM permissions